assets/js/Bible.js

```diff
@@ -244,6 +248,65 @@ class Bible extends Component {
         this.setState({}, this.loadTranslationsAndBooks);
     }
 
+    nextChapter() {
+        // Note: parseInt is here because sometimes selectedChapter is a string.
+        //    Probably when chapter is parsed from the URL during first load it become a string
+
+        const index = this.state.chapters.findIndex(
+            (key) => key === parseInt(this.state.selectedChapter)
+        );
+
+        if (typeof this.state.chapters[index + 1] === "undefined") {
+            this.nextBook();
+            return;
+        }
+        this.changeSelectedChapter(this.state.chapters[index + 1]);
+    }
+
+    prevChapter() {
+        // Note: parseInt is here because sometimes selectedChapter is a string.
+        //    Probably when chapter is parsed from the URL during first load it become a string
+        const index = this.state.chapters.findIndex(
+            (key) => key === parseInt(this.state.selectedChapter)
+        );
+
+        if (index === 0) {
+            this.prevBook();
+            return;
+        }
+
+        this.changeSelectedChapter(this.state.chapters[index - 1]);
+    }
+
+    nextBook() {
+        const keys = Object.keys(this.state.books);
+
+        const index = keys.findIndex(
+            (bookKey) => bookKey === this.state.selectedBook
+        );
+
+        // Note: If the last book is selected one - do nothing
+        if (typeof this.state.structure[keys[index + 1]] === "undefined") {
+            return;
```

Comment:
Can we mark button as inactive in that case? I added dedicated class for that purpose, it's called `.icon-navigator-disabled`. It will block all `click` attempts and change fill color to indicate it was blocked. This should improve UX.